packages/app-mobile/components/SideMenu.tsx

```diff
@@ -0,0 +1,73 @@
+const React = require('react');
+const { connect } = require('react-redux');
+const SideMenu_ = require('react-native-side-menu-updated').default;
+import { Dimensions, DrawerLayoutAndroid, Platform } from 'react-native';
+import { State } from '@joplin/lib/reducer';
+import { FunctionComponent, useEffect, useRef } from 'react';
+
+class SideMenuFallback extends SideMenu_ {
+	public onLayoutChange(e: any) {
+		const { width, height } = e.nativeEvent.layout;
+		const openMenuOffsetPercentage = this.props.openMenuOffset / Dimensions.get('window').width;
+		const openMenuOffset = width * openMenuOffsetPercentage;
+		const hiddenMenuOffset = width * this.state.hiddenMenuOffsetPercentage;
+		this.setState({ width, height, openMenuOffset, hiddenMenuOffset });
+	}
+}
+
+const SideMenuComponent: FunctionComponent<any> = ({
```

Comment:
We avoid `any` as much as possible so please could you define a type for the props? To do that we generally add a `interface Props { ... }` at the top of the file